aliyun/src/test/java/org/apache/iceberg/aliyun/oss/AliyunOSSTestBase.java

```diff
@@ -33,14 +34,20 @@
   private final String bucketName = OSS_TEST_RULE.testBucketName();
   private final String keyPrefix = OSS_TEST_RULE.keyPrefix();
 
+  private OSSFileIO fileIO;
```

Comment:
Why not move the OSSFileIO initialization to [here](https://github.com/apache/iceberg/pull/3553/files#diff-39d04fa07c6ca88cd781acc081580ce62cd460d19327e748e6995cefd073873fR51),  because in my view, it's recommend to initialize the OSSFileIO by using the catalog properties rather than the OSSFIleIO constructor(for the most of the cases).